<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleDebugIOPage</class>
 <widget class="QWidget" name="VehicleDebugIOPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>460</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border:3px solid;
border-color:rgb(230, 230, 230);
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_MC1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border:3px solid;
border-color:rgb(230, 230, 230);
</string>
   </property>
   <property name="text">
    <string>MC1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_T">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border:3px solid;
border-color:rgb(230, 230, 230);
</string>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_MC2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border:3px solid;
border-color:rgb(230, 230, 230);
</string>
   </property>
   <property name="text">
    <string>MC2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Page">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
background-color: rgb(0, 0, 0);
color:white;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_nextpage">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
/*border-color: rgb(216, 216, 216);*/
</string>
   </property>
   <property name="text">
    <string>▼</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_prepage">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);
/*border:solid 1px;*/</string>
   </property>
   <property name="text">
    <string>▲</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>668</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btn_back</zorder>
  <zorder>btn_MC1</zorder>
  <zorder>btn_T</zorder>
  <zorder>btn_MC2</zorder>
  <zorder>lbl_Page</zorder>
  <zorder>btn_nextpage</zorder>
  <zorder>btn_prepage</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
